entoas/generator.go

```diff
@@ -503,25 +503,28 @@ func property(f *gen.Field) (*ogen.Property, error) {
 	return ogen.NewProperty().SetName(f.Name).SetSchema(s), nil
 }
 
-var _types = map[string]*ogen.Schema{
-	"bool":      ogen.Bool(),
-	"time.Time": ogen.DateTime(),
-	"string":    ogen.String(),
-	"[]byte":    ogen.Bytes(),
-	"uuid.UUID": ogen.UUID(),
-	"int":       ogen.Int(),
-	"int8":      ogen.Int32(),
-	"int16":     ogen.Int32(),
-	"int32":     ogen.Int32(),
-	"uint":      ogen.Int32(),
-	"uint8":     ogen.Int32(),
-	"uint16":    ogen.Int32(),
-	"uint32":    ogen.Int32(),
-	"int64":     ogen.Int64(),
-	"uint64":    ogen.Int64(),
-	"float32":   ogen.Float(),
-	"float64":   ogen.Double(),
-}
+var (
+	min    int64
+	_types = map[string]*ogen.Schema{
```

Comment:
Why the _?

Reply:
Good question. When I wrote it there was a lot of Dart code on my desk as well, you unexport members by "_" there.